gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -39,6 +41,8 @@ local function VoiceTryDisable()
 end
 
 local function VoiceTeamTryEnable()
+	if not hook.Run("TTT2CanUseTeamVoiceChat", LocalPlayer()) then return end
```

Comment:
Put the `local ply = LocalPlayer()` above this and use ply instead of calling `LocalPlayer()` twice. 
Also shouldn't this hook be on the serverside?

Reply:
maybe the variable should be called `client` to be in line with our coding style?

Reply:
Done